### PR TITLE
Added more integration tests

### PR DESCRIPTION
**Pull request by keradus**

### CODE REVIEW

File: Symfony/CS/Fixer/Symfony/UnusedUseFixer.php [hunk 1]
```diff
@@ -53,7 +53,9 @@ public function getPriority()
     public function supports(\SplFileInfo $file)
     {
         // some fixtures are auto-generated by Symfony and may contain unused use statements
-        if (false !== strpos($file, DIRECTORY_SEPARATOR.'Fixtures'.DIRECTORY_SEPARATOR)) {
+        if (false !== strpos($file, DIRECTORY_SEPARATOR.'Fixtures'.DIRECTORY_SEPARATOR) &&
+            false === strpos($file, DIRECTORY_SEPARATOR.'CS'.DIRECTORY_SEPARATOR.'Tests'.DIRECTORY_SEPARATOR.'Fixtures'.DIRECTORY_SEPARATOR)
```
Comment (keradus) [inline Symfony/CS/Fixer/Symfony/UnusedUseFixer.php]: to be reverted

**Comment by keradus** [comment]
@SpacePossum I just do some minor change to your code, please take a look ;)

**Comment by keradus** [comment]
Any more tests are welcome!

**Comment by keradus** [comment]
Thank you @SpacePossum.

**Comment by SpacePossum** [comment]
thank you @keradus :)

**Pull request merged**